src/icp_canister_backend/src/lib.rs

```diff
@@ -8,7 +8,7 @@ use std::cell::RefCell;
 
 pub const EPISODE_DURATION: u64 = 91 * 24 * 60 * 60 / 3;
 const MAX_ACTIVE_EPISODES: u64 = 24;
-
+const EPISODE_PROCESSING_INTERVAL: u64 = 60 * 60; 
```

Comment:
The processing interval is too short, can you make the timer dynamic to calculate the exact time for the timer spawn?
So, it starts the first time, it is may be 3 days to the end, then 31 days, etc.